lisa/sut_orchestrator/azure/common.py

```diff
@@ -281,6 +282,14 @@ def get_storage_client(
     )
 
 
+def get_resource_management_client(
+    credential: Any, subscription_id: str
+) -> ResourceManagementClient:
+    return ResourceManagementClient(
```

Comment:
This is a one line method. What's the reason, it needs to be a method?

Reply:
I was following the pattern similar to methods such as `get_storage_client`, `get_network_client` which are defined in common.py. I assume these methods have been encapsulated to hide the internal initialization details as they are used across the project.